### PR TITLE
Update copyrights

### LINKED ISSUE #12419: Copyrights Should be Updated to 2020
io.openliberty.jakarta.el.4.0/bnd.bnd 
io.openliberty.jakarta.el.4.0/src/com/ibm/wsspi/el/ELFactoryWrapperForCDI.java
io.openliberty.jakarta.jsp.3.0/bnd.bnd
io.openliberty.jakarta.jstl.2.0/bnd.bnd
... and others

### PR DESCRIPTION
**Pull request by volosied**
fixes #12419

### CODE REVIEW

File: dev/com.ibm.ws.webcontainer/bnd.bnd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2017 IBM Corporation and others.
+# Copyright (c) 2017,2020 IBM Corporation and others.
 # All rights reserved. This program and the accompanying materials
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer/bnd.bnd:3]: space after comma please

**Comment by volosied** [comment]
#run-libby-bot

**Comment by volosied** [comment]
#run-libby-bot

**Comment by volosied** [comment]
#run-libby-bot

**Pull request merged**